partner_firstname/models/partner.py

```diff
@@ -1,26 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
IMHO, it's easier to define filename from a functional point of view, i.e.: product, partner. Easy to read and allow to gather several models in the same file if there is not a lot features. Keep It Simple please to have a progressive learning curve.
The technical point of view is in the file :-)